src/program.js

```diff
@@ -67,6 +68,11 @@ export class Program {
     name: string, description: string, executor: Function,
     commandOptions: Object = {}
   ): Program {
+    if (!this.defaultValues) {
+      this.defaultValues = {};
```

Comment:
Do this in the [constructor()](https://github.com/shubheksha/web-ext/blob/6761b989e41f55a22ce6789f966dd92392729b76/src/program.js#L43) instead.